pkg/storage/storage.go

```diff
@@ -60,7 +61,7 @@ func IsTooManyClientError(err error) bool {
 	return IsErrorCode(err, TooManyClient)
 }
 
-type Store interface {
+type API interface {
```

Comment:
Why did you need to add this interface?
I find the naming confusing, as this interface is a collection of storage methods.
`APIStore` maybe?

Reply:
The API interface is just to have an interface without Close() and other methods available on the store, as those doesn't make sense on sql transaction context.

Reply:
Can be renamed as well.